ledger/metrics/src/main/scala/com/daml/metrics/InstrumentedExecutorServiceForDocs.scala

```diff
@@ -0,0 +1,55 @@
+// Copyright (c) 2022 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package com.daml.metrics
+
+import com.daml.metrics.MetricDoc.MetricQualification.Debug
+import com.daml.metrics.MetricHandle.{Counter, Meter, Timer}
+
+class InstrumentedExecutorServiceForDocs(name: MetricName) {
```

Comment:
How do we ensure this actually gets updated if new metrics get added to the instrumented part?

Reply:
We do not ensure this.

Reply:
Do we want to ensure it? If so I think since the metrics are private in `InstrumentedExecutorService` we should provide our own class.

Reply:
Not sure, I was just mentioning it because I known that the 4.2.x dropwizard version includes a lot more metrics for the executor service, so I was wondering if at least we have some checklist if we update the versions of those libs, to make sure the metric documentation is up to date.

Reply:
I do not know whether such a checklist exists, sounds like a good idea.

Reply:
@mziolekda

Reply:
I think the best would be to implement an automated check that all metrics have a corresponding documentation and if not, to fail the build

Reply:
I will create a corresponding task.